fetch.bs

```diff
@@ -7178,6 +7174,22 @@ returns failure or a <a for=/>MIME type</a>.
 
 <hr>
 
+<div algorithm>
+<p>To <dfn id=concept-body-get-mime-type for=Body>get the MIME type</dfn>, given a {{Request}}
+or {{Response}} object, <var>requestOrResponse</var>, run these steps:
+
+<ol>
+ <li><p>Let <var>headers</var> be null
+ <li><p>If <var>requestOrResponse</var> is a {{Request}} object, then set <var>headers</var> to
+ <var>requestOrResponse</var>'s <a for=Request>request</a>'s {{Request/headers}}.
```

Comment:
The last one here needs to be `<a for=request>header list</a>`. Similar error for response below.